File: MovieDB.Api/Entities/Theater.cs
using System;
using System.Text.Json.Serialization;

namespace MovieDB.Api.Entities
{
    public enum TheaterGenre
    {
        Opera,
        Operette,
        Musical,
        Theater,
        Ballet
    }

    public class Theater
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public DateTime SeenAt { get; set; }
        public string Location { get; set; }
        public TheaterGenre Genre { get; set; }
        public Rating Rating { get; set; }
        [JsonIgnore]
        public Account Account { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public DateTime? DeletedAt { get; set; }
    }
}

File: MovieDB.Client/src/types/theater.ts
export type TTheater = {
  id: number
  title: string
  seenAt: string
  location: string
  genre: number
  rating: number
}

export const theaterGenres = [
  { text: 'Opera', value: 0 },
  { text: 'Operette', value: 1 },
  { text: 'Musical', value: 2 },
  { text: 'Theater', value: 3 },
  { text: 'Ballet', value: 4 },
]
